packages/commonwealth/client/scripts/models/Thread.ts

```diff
@@ -1,11 +1,116 @@
 import { ProposalType } from 'common-common/src/types';
-import type moment from 'moment';
-import type { VersionHistory } from '../controllers/server/threads';
-import type { IUniqueId } from './interfaces';
-import type Poll from './Poll';
+import type ChainEntity from 'models/ChainEntity';
+import Poll from 'models/Poll';
+import type momentT from 'moment';
```

Comment:
Any reason behind this change (adding the explicit type import rather than using the namespace)?

Ideally if we're going to do it this way, we should use `import type { Moment } from 'moment';` so we don't need to do `momenT.Moment`.

Reply:
Hm, i think there were many commits on this file and I just renamed it from `import type moment from 'moment';` to `import type momentT from 'moment';` because of another similar import. Will fix this